docs/source/usage.rst

```diff
@@ -351,6 +351,35 @@ You can use literal commas and spaces anywhere by escaping (:code:`\,` and :code
     >>> c.escape_whitespace
     'Hello World'
 
+Nested interpolations
+^^^^^^^^^^^^^^^^^^^^^
```

Comment:
same feedback as the one for the notebook:
Nested interpolations applies to all interpolations, they should be documented before custom interpolations.
I am a bit torn about mentioning nested interpolations in custom interpolations once we have a general description.

Also use the term "nested interpolations" or "interpolations can be nested".

Reply:
Changed in 6339fcd

For now I kept the example in custom interpolations as personally I think it's a typical use case that's worth highlighting.